core/src/autogluon/core/_setup_utils.py

```diff
@@ -20,7 +20,7 @@
     'pandas': '>=1.2.5,<1.4',  # Capped version of pandas to 1.4.0 because of issue: https://github.com/pandas-dev/pandas/issues/45603
```

Comment:
```suggestion
    'pandas': '>=1.2.5,!=1.4.0',
```
patch included in 1.4.1 ref https://github.com/pandas-dev/pandas/commit/96cf851e96c9bcca8bcee6138a49e876f3ef12b5